hypha/apply/users/utils.py

```diff
@@ -50,3 +51,36 @@ def send_activation_email(user, site=None):
     subject = ''.join(subject.splitlines())
     message = render_to_string('users/activation/email.txt', context)
     user.email_user(subject, message, settings.DEFAULT_FROM_EMAIL)
+
+
+def send_confirmation_email(user, token, updated_email=None, site=None):
+    """
+    Send the confirmation email. The confirmation token is the update email,
+    signed using TimestampSigner.
+    """
+
+    uid = urlsafe_base64_encode(force_bytes(user.pk))
+
+    activation_path = reverse('users:confirm_email', kwargs={'uidb64': uid, 'token': token})
+
+    context = {
+        'user': user,
+        'name': user.get_full_name(),
+        'username': user.get_username(),
+        'unverified_email': updated_email,
+        'activation_path': activation_path,
+        'timeout_days': settings.EMAIL_CHANGE_CONFIRMATION_HOURS,
```

Comment:
Maybe use PASSWORD_RESET_TIMEOUT_DAYS, it is a Django setting that we already use in other places.